lib/tasks/demo_data_for_development.rake

```diff
@@ -0,0 +1,70 @@
+namespace :data do
+  desc 'Create demo data for our local development'
+  task test: :environment do
+    include FactoryGirl::Syntax::Methods
+
+    def generate_events program, conference
+        user1 = create(:user)
+        user2 = create(:user)
+
+        selected_schedule = create(:schedule, program: program)
+        demo_schedule = create(:schedule, program: program)
+        program.update_attributes(selected_schedule: selected_schedule)
+
+        build(:event, program: program, title: 'Demo Event', abstract: 'This is a demo event instance whose state not defined.').save(validate: false)
```

Comment:
Why do you do `save(validate: false)`? All the objects created here should be valid

Same for all the `events` and `scheduled_events`